community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,118 @@
-agreement after 8 days have passed since a proposal was accepted by all present
-members. An absent member may appoint a single delegate during absences.
+Members of the committee may step down at any time. When a member steps down, the
+company who holds the seat shall nominate a new individual for the seat.
 
-<!-- TODO ## Committee Meeting -->
+Changes to the number of seats, which company seats are allocated to, and nominations
+to the committee are confirmed by majority vote of the committee members.
 
-## Committee Members
+## Frequently asked questions
 
-The members of the Steering Committee are shown below. Membership in the SC is
-determined by current level of contribution to the project. Contribution is
-periodically reviewed to ensure proper recognition.
+_Why are seats company held?_
 
-| &nbsp;                                                   | Member         | Company | Profile                                  |
-| -------------------------------------------------------- | -------------- | ------- | ---------------------------------------- |
-| <img width="30px" src="https://github.com/dewitt.png">   | DeWitt Clinton | Google  | [@dewitt](https://github.com/dewitt)     |
-| <img width="30px" src="https://github.com/mchmarny.png"> | Mark Chmarny   | Google  | [@mchmarny](https://github.com/mchmarny) |
-| <img width="30px" src="https://github.com/isdal.png">    | Tomas Isdal    | Google  | [@isdal](https://github.com/isdal)       |
+The committee was created as the project was in its infancy, in order to tackle governance
+and overall project strategy. Because of the nature of the project and funding required,
+it was decided that strong corporate leadership was necessary for the project to ensure
+velocity.
+
+_What happens if someone leaves the company they represent?_
+
+In that case, the company will nominate a new representative for the seat. The nominee
+shall be confirmed by the standard process as called out above.
+
+_What happens if someone leaves the project or decides to leave the committee?_
+
+As above, the same would apply. The company who holds the seat will nominate a new
+representative to hold the seat. The nominee shall be confirmed by the standard process,
+called out above.
+
+_Is this steering model permanent?_
+
+The governance of Knative is an open, living document, and will continue to evolve as
+the community and project change. We expect over time we will adapt the way we run this
+committee, based on feedback from the community.
+
+_How are steering committee decisions reached?_
+
+The steering committee desires to always reach consensus. Decisions are made in
+meetings when a quorum of the members are present and may pass with majority
+vote. Quorum is considered reached when at least half of the members are present.
+
+The committee's charter may be changed by majority consent of committee members.
+
+In case of extended absence, an absent member will be considered to be in
```

Comment:
I'm a bit unclear about what this is trying to convey. Is it meant to be a rule about who is considered to be absent, or is it a rule about being considered to agree if no objection is given? Or both?

If the former, I am not sure what is gained. An absent person is absent. A rule might be that repeated and/or extended absence triggers the replacement process automatically.

For the latter, I don't think consent by silence is a good rule. If there is an automatic inference, it should be for abstention. Better yet, just record the absence.

Reply:
This is unclear in the current iteration, but my intention was to allow for consent by silence.

Reply:
I don't agree at all that silence is consent. Nulls are not a meaningful signal. If the problem is people disagreeing after a vote is closed, that's on them for not being involved. If the problem is inactive members, being able to remove sitting members for failing to attend a certain number properly-noticed meetings without apologies would work.